appveyor_build.sh

```diff
@@ -81,8 +81,8 @@ run "make opt.opt" make opt.opt
 
 cd ../build-mingw32
 
-cp config/m-nt.h config/m.h
-cp config/s-nt.h config/s.h
+cp config/m-nt.h byterun/caml/m.h
+cp config/s-nt.h caml/byterun/s.h
```

Comment:
This looks like the source of the AppVeyor failure...

Reply:
The branch 4.06 [still fails on AppVeyor](https://ci.appveyor.com/project/Chris00/ocaml-appveyor/build/1.0.213#L327) because it seems this was not properly fixed.